sample_files/pre-commit-13.0/.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
```

Comment:
As I understand the [spec](https://editorconfig-specification.readthedocs.io/en/latest/#file-processing) it might be better to *not* set root so users can have other global preferences? Knowing that .editorconfig in the OCA repo will still have precedence.

Reply:
It means that the engine shouldn't go more up the tree to find further specs.

I thought that would make sense, as user configs could override OCA ones, and I guess what we want is make everyone's IDEs autoconfigured equally, so I thought this would be better like it is.

However you can remove it and we can add it later if we see there's some bug related to it.

Reply:
Thinking it twice, I think this is OK as it is.

Imagine you have a `~/.editorconfig` file like this:

```ini
[*.js]
indent_size = 2
```

As the file selector is more specific, it could override our default, and result in your PRs being wrongly published.

It's better to leave `root = true` so we ensure the engine stops considering `.editorconfig` files in this folder.

Reply:
Perhaps nitpicking but... the spec says "If multiple EditorConfig files have matching sections, the rules from the closer EditorConfig file are read last, so properties in closer files take precedence.", so I think we must remove `root=true` and make an explicit list instead of `[*]` so if someone defines a rule in `~/.editorconfig` for, says .sql files for which OCA as not convention, then his/her definition will be preserved, whereas with `root=true` it will be ignored and with `[*]` it will be overridden.

Not tested myself, I'm just reading the spec.

Reply:
> As the file selector is more specific, it could override our default,

As I read the spec it will not, since the closest `.editorconfig` has priority.